cdc/owner/changefeed.go

```diff
@@ -321,10 +340,28 @@ func (c *changefeed) releaseResources(ctx cdcContext.Context) {
 	c.redoManagerCleanup(ctx)
 	canceledCtx, cancel := context.WithCancel(context.Background())
 	cancel()
-	// We don't need to wait sink Close, pass a canceled context is ok
-	if err := c.sink.close(canceledCtx); err != nil {
-		log.Warn("Closing sink failed in Owner", zap.String("changefeed", c.state.ID), zap.Error(err))
-	}
+
+	// when try to close the kafka sink, it would be blocked on network IO for about 1 minutes.
+	// this could happen for bad network connection between kafka producer and brokers.
```

Comment:
```suggestion
	// When we try to close the Kafka sink, it could be blocked for network IO for about 1 minutes.
	// This could happen due to bad network connection between the Kafka producer and the brokers.
```